server/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import cors from 'cors';
+import { dalleRouter } from './routes/dalle.routes';
+import type { Express } from 'express';
+import express from 'express';
+import { PORT } from './utils/constants';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
This should be removed and use the constants providers by the utils section instead